molo/surveys/forms.py

```diff
@@ -252,13 +252,27 @@ def clean(self):
                               'Options: a True and False'),
                         )
                 elif data['field_type'] in VALID_SKIP_LOGIC:
+                    choice_max_length = 0
```

Comment:
I think this variable name is incorrect. It seems to be the length of all the choices added, not the maximum possible length. Maybe something like `choices_length` would be more accurate?